src/rpc/masternode.cpp

```diff
@@ -252,6 +253,7 @@ UniValue listmasternodes(const UniValue& params, bool fHelp)
             obj.push_back(Pair("network", strNetwork));
             obj.push_back(Pair("txhash", strTxHash));
             obj.push_back(Pair("outidx", (uint64_t)oIdx));
+            obj.push_back(Pair("pubkey", mn->pubKeyMasternode.GetHex()));
```

Comment:
As previously mentioned, looks like using `.GetHex()` is producing some ugly output.

recommend changing to
```c++
obj.push_back(Pair("pubkey", HexStr(mn->pubKeyMasternode)));
```
or, alternatively the format suggested by @random-zebra, whichever format you decide on using for zDEX.